bucket/speedtest-cli.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "1.0.0",
+    "description": "Internet connection speed and performance tester.",
+    "homepage": "https://www.speedtest.net/apps/cli",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.speedtest.net/about/eula"
+    },
+    "url": "https://dl.bintray.com/ookla/download/ookla-speedtest-1.0.0-win64.zip",
```

Comment:
Better to specify the 64bit architecture.

Reply:
```json
{
    "homepage": "https://www.speedtest.net/apps/cli",
    "description": "Speedtest CLI brings the trusted technology and global server network behind Speedtest to the command line.",
    "license": {
        "identifier": "Freeware",
        "url": "https://www.speedtest.net/about/eula"
    },
    "version": "1.0.0",
    "architecture": {
        "64bit": {
            "url": "https://dl.bintray.com/ookla/download/ookla-speedtest-1.0.0-win64.zip",
            "hash": "64054a021dd7d49e618799a35ddbc618dcfc7b3990e28e513a420741717ac1ad"
        }
    },
    "bin": "speedtest.exe",
    "checkver": {
        "regex": "speedtest-([\\d.]+)-win64.zip"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://dl.bintray.com/ookla/download/ookla-speedtest-$version-win64.zip",
                "hash": {
                    "url": "$url.sha256"
                }
            }
        }
    }
}
```

Reply:
Thank you